Add "Resource Recommendations"

davidhammaker opened the pull request:
This PR adds the "Resource Recommendations" article, plus a few very minor tweaks to two of the other articles.

ndirig commented on _posts/2024-01-02-resource-recommendations.html:
```diff
@@ -0,0 +1,200 @@
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
+    <img src="{{ "/assets/images/articles/resource_the-five-disfunctions.jpeg" |
```
The image width should be the same as the article body text.

From design:
![Screenshot 2024-04-17 at 9 39 35 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/627c24a3-30ca-4536-8b24-5750693ac8b1)

vs:
<img width="885" alt="Screenshot 2024-04-17 at 9 40 18 AM" src="https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/490860c6-e828-4ad3-8f84-9be0e0fe9b3f">

ndirig replied:
I now realize these comments are under the wrong image, in the wrong file (this adjustment will need to be in _layouts/article.html, line 17).

This change would affect all other articles, so hopefully this is an easy fix across the board!

ndirig commented on _posts/2024-01-02-resource-recommendations.html:
```diff
@@ -0,0 +1,200 @@
+---
+author:
+duration: 4 Min Read
```
Should be "Upstream"
![Screenshot 2024-04-17 at 9 42 11 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/9a9a03fe-7cc2-4c31-8fcc-8548ea6aa810)

ndirig commented on _posts/2024-01-02-resource-recommendations.html:
```diff
@@ -0,0 +1,200 @@
+    a Team: A Leadership Fable">
+  </div>
+  <div class="content-box order-2 col-12 col-md-7 resource-detail">
+    <p class="resource-topic">Team DNA and Community</p>
```
This should use overline font styling.
![Screenshot 2024-04-17 at 9 47 06 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/ac55d141-563d-4470-a619-04eb0498fd86)

vs
<img width="435" alt="Screenshot 2024-04-17 at 9 48 12 AM" src="https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/fa46c819-758c-4b9a-a38e-d8bedf7d9390">

ndirig replied:
Same comments apply to other resources

ndirig commented on _posts/2024-01-02-resource-recommendations.html:
```diff
@@ -0,0 +1,200 @@
+</p>
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
```
The padding between the photo and the text could be a bit more.

From design:
![Screenshot 2024-04-17 at 9 50 26 AM](https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/c0f2b34c-cd0f-451f-83aa-d6615ee13088)

vs:
<img width="838" alt="Screenshot 2024-04-17 at 9 49 19 AM" src="https://github.com/UpstreamCode/upstreamcode.github.io/assets/31777263/88547f09-ea2b-4a85-8f9e-bb12f66d0295">

ndirig reviewed (changes requested):
Just a few minor requests. This looks great otherwise!

davidhammaker commented:
> Just a few minor requests. This looks great otherwise!

I've added the author, updated the font, and fixed the padding! I didn't touch the article layout yet -- I figure that might be part of a different PR that more comprehensively covers style fixes across the site. Let me know if this is good to go!
